src/styles/block-overrides.scss

```
@@ -0,0 +1,18 @@
+@media (min-width: $break-small) {
+
+	.is-style-wp-block-navigation-separator {
+
+		.wp-block-navigation__responsive-container:not(.is-menu-open) {
+
+			.wp-block-navigation__container.block-editor-block-list__layout {
+				column-gap: 10px !important;
+			}
+
+			.wp-block-navigation-item:not(:last-child) a {
```

```suggestion
			.wp-block-navigation-item:not(:last-child) {
```
I would do this because right now if you hover over the links, the a tag stretches all the way to the border on the right, but not on the left. If you drop the a here then it seems to be properly centered between both borders.